src/gui/torrentcategorydialog.ui

```diff
@@ -6,16 +6,16 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>400</width>
-    <height>100</height>
+    <width>493</width>
+    <height>208</height>
```

Comment:
revert it?

Reply:
> revert it?

Size is changed due to new dialog layout (and new widgets added).